tests/attacks/test_iterative_method.py

```diff
@@ -47,10 +47,10 @@ def setUpClass(cls):
 
         cls.n_train = 100
         cls.n_test = 11
-        cls.x_train_mnist = cls.x_train_mnist[0 : cls.n_train]
-        cls.y_train_mnist = cls.y_train_mnist[0 : cls.n_train]
-        cls.x_test_mnist = cls.x_test_mnist[0 : cls.n_test]
-        cls.y_test_mnist = cls.y_test_mnist[0 : cls.n_test]
+        cls.x_train_mnist = cls.x_train_mnist[0: cls.n_train]
+        cls.y_train_mnist = cls.y_train_mnist[0: cls.n_train]
+        cls.x_test_mnist = cls.x_test_mnist[0: cls.n_test]
+        cls.y_test_mnist = cls.y_test_mnist[0: cls.n_test]
```

Comment:
Please revert these changes because they are not following Black style formatting.
```suggestion
        cls.x_train_mnist = cls.x_train_mnist[0 : cls.n_train]
        cls.y_train_mnist = cls.y_train_mnist[0 : cls.n_train]
        cls.x_test_mnist = cls.x_test_mnist[0 : cls.n_test]
        cls.y_test_mnist = cls.y_test_mnist[0 : cls.n_test]
```